File: Code/apps/Wroker_Server/drivers/python/driver.py
# --- DRIVER CODE START ---
import sys

DELIMITER = "$$$DELIMITER$$$"

try:
    with open("/app/input.txt", "r") as f:
        input_data = f.read()

    test_cases = input_data.split(DELIMITER)

    for test_case in test_cases:
        if not test_case.strip():
            continue

        # For simple problems (e.g. "10 20"), this works.
        # For array inputs (e.g. "[1,2]"), you may need eval() or json.loads() later.
        args = list(map(int, test_case.strip().split()))

        if "solve" not in globals() or not callable(solve):
            print("Function 'solve' is not defined! Did you change the function name?")
            sys.exit(1)

        result = solve(*args)

        print(result)
        print(DELIMITER)

except Exception as err:
    print("Driver Error:", str(err))
    sys.exit(1)

# --- DRIVER CODE END ---

File: Code/apps/Wroker_Server/drivers/javascript/driver.js
// --- DRIVER CODE START ---
const fs = require('fs'); // <--- CHANGED THIS LINE (Use require, not import)

const DELIMITER = '$$$DELIMITER$$$';

try {
    const input = fs.readFileSync('/app/input.txt', 'utf-8');
    const testCases = input.split(DELIMITER);

    testCases.forEach((testCase) => {
        if (!testCase.trim()) return; 

        // For simple problems (e.g. "10 20"), this works.
        // For array inputs (e.g. "[1,2]"), you might need JSON.parse() later.
        const args = testCase.trim().split(/\s+/).map(Number);
        
        if (typeof solve !== 'function') {
            console.error("Function 'solve' is not defined! Did you change the function name?");
            process.exit(1);
        }

        const result = solve(...args);

        console.log(result);
        console.log(DELIMITER);
    });

} catch (err) {
    console.error("Driver Error:", err.message);
    process.exit(1);
}
// --- DRIVER CODE END ---